loading.go

```diff
@@ -88,6 +118,113 @@ func (rs *RiveScript) LoadDirectory(path string, extensions ...string) error {
 	return nil
 }
 
+/*
+LoadHttpFilesystem loads multiple RiveScript documents from a folder on disk.
+
+Parameters
+
+	path: Path to the directory on disk
+	extensions...: List of file extensions to filter on, default is
+	               '.rive' and '.rs'
+*/
+func (rs *RiveScript) LoadHttpFilesystem(httpfs http.FileSystem, path string, extensions ...string) error {
+	if len(extensions) == 0 {
+		extensions = []string{".rive", ".rs"}
+	}
+
+	files, err := vfspath.Glob(httpfs, fmt.Sprintf("%s/*", path))
+	if err != nil {
+		return fmt.Errorf("Failed to open folder %s: %s", path, err)
+	}
+
+	// No files matched?
+	if len(files) == 0 {
+		return fmt.Errorf("No RiveScript source files were found in %s", path)
+	}
+
+	var anyValid bool
+	for _, f := range files {
+		// Restrict file extensions.
+		validExtension := false
+		for _, exten := range extensions {
+			if strings.HasSuffix(f, exten) {
+				validExtension = true
+				break
+			}
+		}
+
+		if validExtension {
+			anyValid = true
+			err := rs.LoadHttpFile(httpfs, f)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	if !anyValid {
+		return fmt.Errorf("No RiveScript source files were found in %s", path)
+	}
+
+	return nil
+}
+
+/*
+LoadMultipleDirectory loads multiple RiveScript documents from a comma separated list of folders on disk.
+
+Parameters
+
+	comma separated list of paths: Paths to the directorys on disk
+	extensions...: List of file extensions to filter on, default is
+	               '.rive' and '.rs'
+*/
+func (rs *RiveScript) LoadMultipleDirectory(paths []string, extensions ...string) error {
```

Comment:
The documentation on this function is inaccurate since `paths` is a string slice and not a comma separated string.

And maybe a more semantic name for this function would be `LoadDirectories()`?